### PR TITLE
Refactor/styling

### LINKED ISSUE #70: Make sentiment indicators uniform across site
Compare:
https://emojipedia.org/neutral-face/
https://emojipedia.org/face-without-mouth/

### LINKED ISSUE #83: Refactor styling
- [x] Align article titles to the left
- [x] Use multi-emoji icon for assessment
- [x] Use history icon for history
- [x] Use all caps for section
- [x] Switch to Open Sans for most fonts

### PR DESCRIPTION
**Pull request by nzambonivergara**
### Features
- NA

### Refactors 
- Closes #70 
- Closes #83 
- Made WellNews logo a link for 'feed/home'
- Added emoji icon to return to the check in question
- Added ARIA labels where material icons are used

### Issues and Technical Debt
- NA

### Other Comments, Concerns, or Questions
- NA

### Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature work
- [x] New and existing unit tests pass locally with my changes
- [x] I ran Lighthouse and WAVE accessibility audits and fixed errors/warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const App = (): JSX.Element => {
       setSavedArticles([...savedArticles, newSavedArticle!])
     } else {
       const newSavedArticles = savedArticles.filter(savedArticle => {
-        savedArticle.id !== id;
```
Comment (anthony-iacono) [inline src/components/App/App.tsx:146]: Good catch!

File: src/components/Header/Header.tsx [hunk 2]
```diff
@@ -2,22 +2,27 @@ import React from 'react';
...
   return (
     <header className="header">
-      <div className="header-elements-container">
-        <Link to='/'>
-          <button className="check-in-btn">
-            <span className="check-in-text">Check In</span>
-          </button>
-        </Link>
+      <Link to='/'>
+        <button className="check-in-btn" aria-label="Return to check in question">
```
Comment (anthony-iacono) [inline src/components/Header/Header.tsx:13]: Good eye for ensuring accessibility 👍

**Review by anthony-iacono — APPROVED** [review]
The CSS looks much DRYer. Glad we could get the app looking solid on mobile!

File: src/components/Header/Header.tsx (hunk 2 above)
Comment (najinl) [reply src/components/Header/Header.tsx:13]: Accessibility 👸🏻

**Review by najinl — APPROVED** [review]

**Pull request merged**